dask/array/tests/test_fft.py

```diff
@@ -108,3 +113,41 @@ def test_fft_consistent_names():
     assert same_keys(fft(darr, 5), fft(darr, 5))
     assert same_keys(fft(darr2, 5, axis=0), fft(darr2, 5, axis=0))
     assert not same_keys(fft(darr, 5), fft(darr, 13))
+
+
+def test_wrap_numpy():
```

Comment:
> Yeah, I wasn't blaming you for style things. Just a request. I'm fine keeping things focused if you prefer.

No worries. I didn't think you were. Maybe we should document a set of things like this that we would like to see improved in a separate issue. As I'll likely be back to muck with this code, I can try to pick it up at some point.